Content.Server/GameTicking/Rules/PiratesRuleSystem.cs

```diff
@@ -216,7 +217,7 @@ private void OnPlayerSpawningEvent(RulePlayerSpawningEvent ev)
                 _mindSystem.SetUserId(newMind, session.UserId);
 
                 var mob = Spawn("MobHuman", _random.Pick(spawns));
-                _metaData.SetEntityName(mob, name);
+                MetaData(mob).EntityName = name;
```

Comment:
why?

Reply:
Hmm, probably made a mistake while I was working on #19582, no idea how it got here.